accepted/2024/[TS]PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,77 @@
+# Markdown Readme Rendering in PM UI
+
+- Jonatan Gonzalez ([jgonz120](https://github.com/jgonz120)) 
+- Issue [#12583](https://github.com/NuGet/Home/issues/12583) <!-- GitHub Issue link -->
+- [Feature Spec](https://github.com/NuGet/Home/blob/7943122dffa435f4daeee600efcc5b744cd2e97e/accepted/2023/PMUI-Readme-rendering.md)
+
+## Summary
+
+We want to add update the PM UI to render the ReadMe.md file while browing packages. 
+
+## Motivation 
+
+We want help our customers understand what a package does and if it will be helpful to them. This could also increase ReadMe adoption if developers know their users will see the ReadMe. 
+
+## Explanation
+
+### Functional explanation
+When a package is selected we will determine if a ReadMe file exists and if it does we'll render it in the PM UI. 
+
+The PM UI will be updated to have tabs for the Package Details and the the ReadMe. This UX will be displayed for both the Browse and installed tabs.
+![Alt text](https://github.com/NuGet/Home/assets/89422562/81b24877-f12f-4783-905c-4a155d3c7693)
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+### Technical explanation
```

Comment:
I'm trying to understand what an error could mean for the renderer. If that means that the MD is malformed then we should see this happening on NuGet.org right? 

Either way I would think the error should be visible in the PMUI since putting it in the Output pane would be confusing especially if it's not already open. What's a gold bar?

Reply:
> Either way I would think the error should be visible in the PMUI since putting it in the Output pane would be confusing especially if it's not already open.

👍 Agreed. In another comment, I mention an error message/retry button.

Reply:
![image](https://github.com/NuGet/Home/assets/5552792/e68ee6ca-8ec0-4b54-a4bd-92e1a78bd5c8)
How does this look?

Reply:
Light Mode
![image](https://github.com/NuGet/Home/assets/5552792/22bb4589-85a9-4890-84ca-3a79973e35f8)

Reply:
And High Contrast bc I'm already here
![image](https://github.com/NuGet/Home/assets/5552792/4704ecda-dcc9-4a00-9d5a-35550b55c7d6)

Reply:
That could work. I'd lean towards my suggestion of a control with icon/text and possibly reload button. As discussed offline, get input from the UX Board on it.